components/clinical/timeline/src/atoms/timeline-time.tsx

```diff
@@ -7,9 +7,33 @@ import {
   TimelineDomainResourceType,
 } from '@ltht-react/types'
 import { formatTime } from '@ltht-react/utils'
+import Icon from "@ltht-react/icon";
+import styled from "@emotion/styled";
 
-const TimelineTime: FC<Props> = ({ domainResource, domainResourceType, ...rest }) => {
+const StyledLeftTimeIcon = styled(Icon)`
```

Comment:
Do we need to consider screen reader ability and use an anchor or link instead? https://www.w3schools.com/accessibility/accessibility_buttons_links.php 
I don't believe the current Time icon will do anything with screen reader in its current form.

Reply:
We sure do, good spot